src/Mod/TechDraw/Gui/ViewProviderDimension.cpp

```diff
@@ -1,5 +1,5 @@
 ﻿/***************************************************************************
- *   Copyright (c) 2004 J�rgen Riegel <juergen.riegel@web.de>              *
+ *   Copyright (c) 2004 J�rgen Riegel <juergen.riegel@web.de>              *

```

Comment:
This seems like it's not using UTF8 codification as the ü in Jürgen is not displayed correctly.